loader.lua

```diff
@@ -883,6 +990,35 @@ local function drawUI()
         font_color     = {r=1, b=1, g=1},
         tooltip        = "Click to load deck from notebook",
     })
+
+    self.createButton({
+        click_function = "onToggleAdvancedButton",
+        function_owner = self,
+        label          = "⚙",
```

Comment:
I love this gear character but for some reason it's just showing up as blank in my game :(
Maybe lets use dots instead, cause those are sure to work for everyone?
```suggestion
        label          = "...",
```

Reply:
Works for me, odd that the gear isn't showing up :(